Gopkg.lock

```diff
@@ -723,6 +724,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "55a5e07cebdd4d6b3c6539e42ff0183417a945369810a995bdda7d3e0166bb36"
+  inputs-digest = "3b54f4cb58cc2cd48c554b8e17ffded5e11d2b4cd62b4b66878758b51e696e02"
```

Comment:
I assume this is a newer version?

Reply:
This is updated on every change in this file.

Reply:
heh, right. I skimmed this PR and saw a hash changing and `dep` 🤦‍♂️